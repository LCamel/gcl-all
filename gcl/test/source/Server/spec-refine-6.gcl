con A, B : Int { A ≥ 0 ∧ B > 0 }

if True ->  [!
            ?
            ?
            !]
fi

{ True }