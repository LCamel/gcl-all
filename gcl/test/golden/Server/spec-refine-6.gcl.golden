### Result

[Display, UpdateSpecs, ResMarkPOs]

### Source

con A, B : Int { A ≥ 0 ∧ B > 0 }

if True ->  [!
            
            !]
            [!
            
            !]

fi

{ True }

### Trace

TraceLog "      [ save ] Parsed"
TraceLog "      [ save ] Converted"
TraceLog "      [ save ] TypeChecked"
TraceLog "      [ save ] Swept"
TraceLog "      [ event ] mute"
TraceLog "      [ save ] Mouse selection: spec-refine-6.gcl [46-91] 3:13-6:15"
TraceGetSource
TraceLog "      [ save ] Parsed"
TraceLog "      [ save ] Converted"
TraceLog "      [ save ] TypeChecked"
TraceLog "      [ save ] Swept"
TraceGetSource
TraceGetSource
TraceLog "      [ edit ] Replace ./test/source/Server/spec-refine-6.gcl [46-91] 3:13-6:15 \"[!\n            ?\n            ?\n            !]\" \n      with \"?\n            ?\n\""
TraceEditText ./test/source/Server/spec-refine-6.gcl [46-91] 3:13-6:15 "?\n            ?\n"
TraceLog "      [ save ] Parsed"
TraceLog "    < DigHole ./test/source/Server/spec-refine-6.gcl [46-47] 3:13-14"
TraceGetSource
TraceLog "      [ edit ] Replace ./test/source/Server/spec-refine-6.gcl [46-47] 3:13-14 \"?\" \n      with \"[!\n            \n            !]\""
TraceEditText ./test/source/Server/spec-refine-6.gcl [46-47] 3:13-14 "[!\n            \n            !]"
TraceLog "      [ save ] Parsed"
TraceLog "    < DigHole ./test/source/Server/spec-refine-6.gcl [89-90] 6:13-14"
TraceGetSource
TraceLog "      [ edit ] Replace ./test/source/Server/spec-refine-6.gcl [89-90] 6:13-14 \"?\" \n      with \"[!\n            \n            !]\""
TraceEditText ./test/source/Server/spec-refine-6.gcl [89-90] 6:13-14 "[!\n            \n            !]"
TraceLog "      [ save ] Parsed"
TraceLog "      [ save ] Converted"
TraceLog "      [ save ] Converted"
TraceLog "      [ save ] Converted"
TraceLog "      [ save ] TypeChecked"
TraceLog "      [ event ] unmute"
TraceLog "      [ save ] Swept"
TraceLog "      [ load ] Mouse selection: Nothing"
TraceLog "    < Send Diagnostics 0"
TraceSendDiagnostics []
